README.md

```diff
@@ -1 +1,68 @@
-# countries-of-the-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Countries Of The World
+
+> A web application that provides information about each country in the world. The App is based on an external ([REST Countries API](https://restcountries.com/)) that allows users to filter these countries by selecting a specific country. And information (details) about this country is displayed, such as population, area, etc.
+
+![preview](./src/assets/mobile.gif)
+
+## Built With
+
+- HTML & CSS
+- JavaScript
+- React, Redux
+- [REST Countries API](https://restcountries.com/)
+
+## Live Demo (If available)
+
+[Live demo link](https://space-travelers-react.herokuapp.com/)
+
```

Comment:
- [ ] <b>REQUIRED:</b> The link here is for for project. Kindly deploy this project and replace this link with the correct one.